### DISCUSSION TITLE
is there a membership style feature

### DISCUSSION THREAD

**Discussion by thomasyit** [squidfunk/mkdocs-material#5395] [Q&A] [answered]
to make pages private based on subscription service?

**Comment by kamilkrzyskow** [comment, marked as answer]
Asked multiple times before, and no, there is no such feature in the theme, you have to add external server side authorization yourself.
- https://github.com/squidfunk/mkdocs-material/discussions/5360
- https://github.com/squidfunk/mkdocs-material/discussions/5050
